doc/about.rst

```diff
@@ -0,0 +1,47 @@
+.. _about:
+
+======================
+About ``mybinder.org``
+======================
+
+This page explains some of the teams and organizations behind mybinder.org.
+
+What technology runs mybinder.org?
+==================================
+
+The technology behind mybinder.org is primarily composed of three open-source projects:
+
+* `JupyterHub <https://z2jh.jupyter.org>`_, which manages cloud infrastructure for user instances
+* `repo2docker <repo2docker.readthedocs.io>`_, which builds Docker images from GitHub repositories
+* `BinderHub <binderhub.readthedocs.io>`_, which orchestrates the above two projects and
+  provides the Binder interface.
+
+Each of these are open and inclusively-governed projects. Currently, these are all officially
+hosted as a part of Project Jupyter, an open project that creates open tools for data science
+infrastructure and interactive computation. The mybinder.org team is
+heavily involved in each, though mybinder.org does not need to have a controlling
+stake in any of them.
+
+Who runs mybinder.org?
+======================
+
+mybinder.org is a single deployment of the BinderHub technology. It is run as a free, public
+service to the scientific and educational community. It also serves as a way for the Binder
+project to test the "cutting edge" of Binder and demonstrate the technology.
+
+Currently, the mybinder.org team is comprised of the `Binder Project team <https://jupyterhub-team-compass.readthedocs.io/en/latest/team.html#binder-team>`_.
+However, as time moves on we expect that these two teams will start to diverge as more
+institutions deploy BinderHub for their own purposes, and want to help govern the direction
+of the project.
+
+What kind of an organization is the mybinder.org team?
+======================================================
+
+The mybinder.org team is currently housed as a member of Project Jupyter, which is
```

Comment:
I think we have a few too many different names for the same team. There is Binder team, Binder project, mybinder.org team :-/

How about using (and then we need to try and get it uniformly applied over this page): mybinder.org is operated by the Binder project (an organisation) which is part of Project Jupyter. The term "Binder team" refers to the humans that make up the Binder project. BinderHub and repo2docker (as an example) are software projects that the Binder team leads and contributes to.

There is no mybinder.org team, Binder project team, BinderHub team or other variations.

Reply:
ok, latest push tries to clean some of that up, what do folks think?